source/direct_debit/index.html.md.erb

```diff
@@ -257,23 +256,31 @@ This would return payments 501 to 1000.
     </tr>
     <tr>
     <td nowrap><code>started</code></td>
-    <td>The user has visited the <code>next_url</code> and landed on the GOV.UK Pay page to enter their account information.</td>
+    <td>Your user has visited the <code>next_url</code> and landed on the GOV.UK Pay page to enter their account information.</td>
+    </tr>
+    <tr>
+    <td nowrap><code>abandoned</code></td>
+    <td>GOV.UK Pay did not set up the mandate, because your user took longer than 90 minutes to finish entering their account information.</td>
+    </tr>
+    <tr>
+    <td nowrap><code>failed</code></td>
+    <td>GOV.UK Pay could not set up the mandate, for example because your user's account information was incorrect or your user's bank account does not support Direct Debit.</td>
```

Comment:
Technically, it’s GoCardless that couldn’t set up the mandate, but I’m not sure if that’s important.

Reply:
Thanks Alex - updated in https://github.com/alphagov/pay-tech-docs/pull/306/commits.